tools-iuc/test-data.yaml

```diff
@@ -0,0 +1,3058 @@
+- data:
+    operation_type|input_reads:
+      filetype: null
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/child.fasta
+      sha: 5223a9704fb4547ed6f1afd7f5da3e2c03531ab2
+      unpack: false
+  doc: Test dataset operation_type|input_reads for meryl.
+  license: MIT
+  size: 1048576
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
```

Comment:
this double name is probably confusing for users of this repo, so mabye we only take the latter ... but then we need to deal with duplicates